bin/plugins/scanner/__init__.py

```diff
@@ -15,12 +15,17 @@
 
 def waiting_server(socket_client, host, port):
     connected = False
+    timeout = time() + 10
     while not connected:
         try:
             socket_client.connect((host, port))
             connected = True
         except:
-            pass
+            sleep(0.001)
+            if time() > timeout:
+                traceback.print_exc()
+            else:
+                pass
```

Comment:
The time of a raise the exception `socketerror` is one second approx. So every second you, substract 1ms al timeout and it becomes very slow. Since you have no control over the exception time, I propose: 7da70157d32090fc14f277a22f75081be8dc2d9d